mappings/net/minecraft/client/render/item/ItemRenderer.mapping

```diff
@@ -93,7 +107,7 @@ CLASS net/minecraft/class_918 net/minecraft/client/render/item/ItemRenderer
 		ARG 2 stack
 		ARG 3 x
 		ARG 4 y
-	METHOD method_4026 renderGuiItem (Lnet/minecraft/class_1309;Lnet/minecraft/class_1799;II)V
+	METHOD method_4026 renderGuiItemInner (Lnet/minecraft/class_1309;Lnet/minecraft/class_1799;II)V
```

Comment:
"inner"? What's this for?

Reply:
It follows the naming convention of `blitInner` (from 1.15.2). It's a private implementation method for `renderGuiItem`. The public one with the same signature (`method_27951`) passes through to this method, as do all the overloads.

Reply:
`doRenderInScreen` `innerRenderInScreen` sounds better. We tend ton use screen over gui and just call it in gui/screen than gui/screen item imo

Reply:
screen is just a type of GUI that... well... takes up your whole screen